org.eclipse.cpacep/src/org/eclipse/cpacep/test/CPACEPConnectorTest.java

```diff
@@ -0,0 +1,53 @@
+package org.eclipse.cpacep.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import org.eclipse.cpacep.util.CPACEPConnector;
+import org.eclipse.cpacep.util.FileHandler;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CPACEPConnectorTest {
+	//String homePath = "/home/supun/Documents/cpachecker";
+	//String homePath = "/home/supun/Documents/CPAchecker-1.7-unix";
+	String homePath = "/home/travis/build/SupunArunoda/CPAChecker/cpachecker";
```

Comment:
This will only work for you.
In addition, try to use the `java.nio.Path` and `java.nio.Paths` classes for specifying paths.
That circumvents any OS-specific issues and makes it easier to work with file system paths.
See 888867c for examples.